axelor-admin/src/main/java/com/axelor/apps/base/service/app/DataBackupCreateService.java

```diff
@@ -307,6 +328,8 @@ private String getMetaModelHeader(Object value, Property property, CSVInput csvI
     switch (propertyTypeStr) {
       case "LONG":
         return propertyName.equalsIgnoreCase("id") ? "importId" : propertyName;
+      case "BINARY":
+        return DataBackupServiceImpl.byteArrFnameColName;
```

Comment:
It will not work when you have more than one byte[] field. Also better to have just prefix to identify it as a byte[] field. Like 'byte_', so if field name is 'image' then column name will be 'byte_image'.